src/runtime/pkg/oci/utils.go

```diff
@@ -475,6 +475,16 @@ func addHypervisorConfigOverrides(ocispec specs.Spec, config *vc.SandboxConfig,
 		}
 	}
 
+	if value, ok := ocispec.Annotations[vcAnnotations.GuestHookTimeout]; ok {
+		if value != "" {
+			timeout, err := strconv.ParseInt(value, 10, 32)
+			if err != nil {
+				return err
+			}
+			config.HypervisorConfig.GuestHookTimeout = int32(timeout)
```

Comment:
I just want to note that if you change the message's `int32` to `uint32`, you should notice this kind of places.